test/unit/llm-events/aws-bedrock/bedrock-command.test.js

```diff
@@ -73,19 +74,20 @@ const titanEmbed = {
   }
 }
 
-tap.beforeEach((t) => {
-  t.context.input = {
+test.beforeEach((ctx) => {
+  ctx.nr = {}
+  ctx.nr.input = {
     body: JSON.stringify('{"foo":"foo"}')
   }
 
-  t.context.updatePayload = (payload) => {
-    t.context.input.modelId = payload.modelId
-    t.context.input.body = JSON.stringify(payload.body)
+  ctx.nr.updatePayload = (payload) => {
+    ctx.nr.input.modelId = payload.modelId
+    ctx.nr.input.body = JSON.stringify(payload.body)
   }
 })
 
-tap.test('non-conforming command is handled gracefully', async (t) => {
-  const cmd = new BedrockCommand(t.context.input)
+test('non-conforming command is handled gracefully', async (ctx) => {
```

Comment:
Agreed that the first parameter passed to a test function is technically a "test context" object, but the pattern we are using everywhere is to name that context `t`.

I think that the `beforeEach` and `afterEach` hooks are more explicitly about setting up the context for each subtest's execution. Therefore, I think using `ctx` as the parameter name for those hooks is clear. But I think `test('something', t => {})` is clearer for the individual tests.